extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryFrameProcessor.java

```diff
@@ -99,7 +105,8 @@ public WindowOperatorQueryFrameProcessor(
       final List<OperatorFactory> operatorFactoryList,
       final RowSignature rowSignature,
       final int maxRowsMaterializedInWindow,
-      final List<String> partitionColumnNames
+      final List<String> partitionColumnNames,
+      final boolean addVirtualColumns
```

Comment:
Can revert this change. LMK if you face any error - correctness or otherwise while running the query after making the suggested changes.

Reply:
Done